mlflow/server/js/src/model-registry/components/ModelView.js

```diff
@@ -111,7 +117,31 @@ export class ModelView extends React.Component {
       });
   };
 
+  // rowSelection object indicates the need for row selection
+  createRowSelection = () => {
+    return {
+      onChange: (selectedRowKeys, selectedRows) => {
+        const newState = Object.assign({}, this.state);
+        newState.runsSelected = {};
+        selectedRows.forEach((row) => {
+          newState.runsSelected = {
+            ...newState.runsSelected,
+            [row.run_id]: row.version,
+          };
+        });
+        this.setState(newState);
+      }
+    };
+  };
+
+  onCompare() {
```

Comment:
To test for ModelView.js:

* Compare button behavior (call `setState` on a shallow-rendered `ModelView` & call onCompare, verify that `props.history.push` is called with appropriate args). This will require mocking out `props.history.push` as a `jest.fn()`. You can see an example of that [here](https://github.com/mlflow/mlflow/blob/branch-1.5/mlflow/server/js/src/components/MetricsPlotPanel.test.js#L68), and examples of how to assert that a `jest.fn` was called with arguments in the [API docs](https://jestjs.io/docs/en/expect#tohavebeencalledwitharg1-arg2-)